Model/Api/Data/AutomatedTesting/Config.php

```diff
@@ -29,6 +29,11 @@ class Config implements \JsonSerializable
      */
     private $cart;
 
+    /**
+     * @var Order
```

Comment:
i think technically this can be null
```suggestion
     * @var Order|null
```

Reply:
cool